training_toolbox/lpr/infer_ie.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
```

Comment:
In SSD we don't use it, so every script is started with `python` explicitly. Could you please change it?

Reply:
Fixed.